ext/Pod-Html/bin/pod2html

```diff
@@ -155,27 +145,19 @@ Specify the base directory for finding library pods.
 =item quiet
 
   --quiet
-
-Don't display mostly harmless warning messages.
-
-=item noquiet
-
   --noquiet
 
-Display mostly harmless warning messages (default behaviour). But this is not
-the same as "verbose" mode.
+Don't display mostly harmless warning messages.  --noquiet -- which is the
+default behavior does not display these mostly harmless warning messages (but
+this is not the same as "verbose" mode).
```

Comment:
The reverses the documented meaning of --noquiet.

Reply:
Thanks for spotting that.  Should be corrected in latest commit.